src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -370,54 +436,62 @@ def main(lux):
         else:
             lux.saveFile()
 
-    scene_file = lux.getSceneInfo()["file"]
-    external_files = lux.getExternalFiles()
+    dialog_selections = options_dialog()
+
+    if not dialog_selections:
+        print("Job submission canceled.")
+        return
```

Comment:
We should eventually raise an exception here so the job status doesn't show Suceess afterwards. Related PR: https://github.com/aws-deadline/deadline-cloud-for-keyshot/pull/102 Whichever PR get's merged first, we should fix the other.